### PR TITLE
Statistics: add notice on max_age - closes #25506

### LINKED ISSUE #25506: Statistics: specify info on max_age & sampling_size
### Feedback

It would be great if we can highlight the importance of setting `sampling_size` even when using `max_age` for the `statistics` integration. See more info at: https://community.home-assistant.io/t/statistics-max-age-and-sampling-size/424166

### URL

https://www.home-assistant.io/integrations/statistics

### Version

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by nathan-gs**
## Proposed change

Add a notice clarifying the need to specify `sampling_size` when using `max_age` in the statistics module. Closing #25506

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #25506 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/statistics.markdown [hunk 1]
```diff
@@ -155,3 +155,8 @@ unique_id:
   required: false
   type: string
 {% endconfiguration %}
+
+| :exclamation: An important note of `max_age` and `sampling_size`t   |
```
Comment (frenck) [inline source/_integrations/statistics.markdown]: This is not how we write notices, please consult our developer documentation for more information on how to write documentation.

Considering the limited message below, I'm not sure if any emphasis is needed at all.

File: source/_integrations/statistics.markdown [hunk 2]
```diff
@@ -155,3 +155,8 @@ unique_id:
...
+|---------------------------------------------------------------------|
+
+The `max_age` variable is only valid within the measured samples specified by `sampling_size` (default 20). Specify a wide-enough `sampling_size` if using an extended max age (eg, when looking for `max_age` 1 hour, a sensor which produces 1 measurement a minute should have at least a `sampling_size` of 60.
```
Comment (frenck) [inline source/_integrations/statistics.markdown]:
```suggestion
The `max_age` option is only valid within the measured samples specified by `sampling_size` (default 20). Specify a wide-enough `sampling_size` if using an extended max-age (e.g., when looking for `max_age` 1 hour, a sensor that produces one measurement a minute should have at least a `sampling_size` of 60.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by nathan-gs** [comment]
@frenck changed

**Review by frenck — APPROVED** [review]
Thanks, @nathan-gs 👍

**Pull request merged**